src/main/java/net/oauth/jsontoken/AbstractJsonTokenParser.java

```diff
@@ -167,10 +164,7 @@ public boolean expirationIsValid(JsonToken jsonToken, Instant now) {
    */
   public boolean issuedAtIsValid(JsonToken jsonToken, Instant now) {
     Instant issuedAt = jsonToken.getIssuedAt();
-    if ((issuedAt != null) && now.isBefore(issuedAt)) {
-      return false;
-    }
-    return true;
+    return issuedAt == null || issuedAt.isAfter(now);
```

Comment:
Tests should be able to catch this once we improve the coverage.
```suggestion
    return issuedAt == null || issuedAt.isBefore(now);
```

Reply:
Fixed